botocore/client.py

```diff
@@ -183,10 +183,13 @@ def _get_client_args(self, service_model, region_name, is_secure,
 
         endpoint_creator = EndpointCreator(self._endpoint_resolver,
                                            region_name, event_emitter)
+        timeout = (client_config.connect_timeout, client_config.read_timeout) \
```

Comment:
I \
don't \
like the "\" either. :-)

I would like to take this chance to hear your comment on the following alternatives. They all pass PEP8.

        # Option 1
        if client_config is None:
            timeout = DEFAULT_TIMEOUT
        else:
            timeout = (client_config.connect_timeout,
                       client_config.read_timeout)

        # Option 2
        timeout = (
            client_config.connect_timeout, client_config.read_timeout
            ) if client_config is not None else Config.DEFAULT_TIMEOUT

        # Option 3: Skip the timeout variable definition, just calculate it and use it on-the-fly
        endpoint = endpoint_creator.create_endpoint(
            ...,
            timeout=DEFAULT_TIMEOUT if client_config is None else (
                client_config.connect_timeout, client_config.read_timeout),
            )

The difference is subtle and probably a matter of taste. Just want to know the team's style so that we are in tune.